src/test/java/bio/terra/service/search/SearchServiceTest.java

```diff
@@ -154,6 +156,26 @@ public void querySnapshotTest() throws Exception {
     assertEquals(expectedResultModel.getResult(), actualResultModel.getResult());
   }
 
+  @Test
+  public void snapshotPreviewTest() throws Exception {
+    final UUID snapshotId = UUID.randomUUID();
+    final String tableName = "mockTable";
+    final int limit = 0;
+    final int offset = 30;
+
+    SnapshotPreviewModel expectedSnapshotPreviewModel =
+        new SnapshotPreviewModel().result(new ArrayList<>(values));
+
+    SnapshotService mockSnapshotService = mock(SnapshotService.class);
+    when(mockSnapshotService.retrievePreview(snapshotId, tableName, limit, offset))
+        .thenReturn(expectedSnapshotPreviewModel);
+
+    SnapshotPreviewModel actualSnapshotPreviewModel =
+        mockSnapshotService.retrievePreview(snapshotId, tableName, limit, offset);
+
+    assertEquals(expectedSnapshotPreviewModel.getResult(), actualSnapshotPreviewModel.getResult());
```

Comment:
Will more test code be added here? This doesn't seem like a very useful test as it's currently written; in general it's not useful to assert on the result of calling a mocked API.

Reply:
I'm not sure what other tests we can add here. Most of the work is done in BigQuery, and that's being tested as well.

Reply:
I think this test can be removed without reducing coverage. To find out, you can run the test in the IDE with coverage and see if any of the lines in `SnapshotService` generate coverage data. If no lines generate coverage data, then this test is only testing the mock and should be removed.

Reply:
If you do keep this code, it should be moved to the test class that corresponds to the one it's testing. If it's testing `SnapshotService`, it should be in `SnapshotServiceTest`.